packages/tokens/src/tokens/color/semantic-light.json

```diff
@@ -1,14 +1,14 @@
 {
-  "semantic-test-on-light": {
-    "name": "semantic-test-on-light",
-    "value": "{uswds-system-color-blue-vivid-60}",
+  "vads-color-fg-default-on-light": {
+    "name": "vads-color-fg-default-on-light",
+    "value": "{vads-color-base.*.value}",
```

Comment:
Assuming I'm correct that this is intended to be used for the text color, were we intentionally changing the color? Looks like this is #1b1b1b while the existing light mode font color is #3d4551 (grayDark).

Reply:
Yup, we're intentionally updating the text color -- this will be updated in all of the components and eventually throughout the app. I discussed the Base vs GrayDark text color with Lauren and Brea and we all agreed that the Base color was a better default -- it provides more contrast with our backgrounds and it's consistent with VADS.